src/views/community/beneficiary/index.tsx

```diff
@@ -310,7 +310,10 @@ function BeneficiaryScreen() {
                             </View>
                         </View>
                         <Claim
-                            claimAmount={community.contract.claimAmount}
+                            claimAmount={community.contract.claimAmount.replace(
```

Comment:
This here will not do any change. The values here are in gwei. Meaning that $0.50 would be 50000000000000000.
See more here https://impactmarket-developers.netlify.app/#/quickstart?id=how-to-treat-monetary-values
This value is then processed in https://github.com/impactMarket/mobile-app/blob/feature/anonymous-report/src/views/community/beneficiary/Claim.tsx#L345 and a few other places.